console/web3ext/web3ext.go

```diff
@@ -828,137 +828,137 @@ web3._extend({
 });
 `
 
-const Klay_JS = `
+const Kaia_JS = `
 var blockWithConsensusInfoCall = function (args) {
-    return (web3._extend.utils.isString(args[0]) && args[0].indexOf('0x') === 0) ? "klay_getBlockWithConsensusInfoByHash" : "klay_getBlockWithConsensusInfoByNumber";
+    return (web3._extend.utils.isString(args[0]) && args[0].indexOf('0x') === 0) ? "kaia_getBlockWithConsensusInfoByHash" : "kaia_getBlockWithConsensusInfoByNumber";
 };
 
 web3._extend({
-	property: 'klay',
+	property: 'kaia',
 	methods: [
 		new web3._extend.Method({
 			name: 'clientVersion',
-			call: 'klay_clientVersion',
+			call: 'kaia_clientVersion',
```

Comment:
1. Can we NOT change the `call` field now? because the new `ken attach` won't work against the old endpoints.
2. Can we have both `klay` and `kaia` on the console?
```js
const klayMethods = `
var klayMethods = [
    new web3._extend.Method({
    // ...
]
`

const Klay_JS = klayMethods + `
web3._extend({ property: 'klay', methods: klayMethods })
`

const Kaia_JS = klayMethods + `
web3._extend({ property: 'kaia', methods: klayMethods })
`
```

Reply:
fixed 893f4ed2 0bba9474